lib/ui/geometry.dart

```diff
@@ -619,66 +619,53 @@ class Size extends OffsetBase {
 /// Rect myRect = const Offset(1.0, 2.0) & const Size(3.0, 4.0);
 /// ```
 class Rect {
-  Rect._();
-
   /// Construct a rectangle from its left, top, right, and bottom edges.
   @pragma('vm:entry-point')
-  Rect.fromLTRB(double left, double top, double right, double bottom) {
-    _value
-      ..[0] = left
-      ..[1] = top
-      ..[2] = right
-      ..[3] = bottom;
-  }
+  const Rect.fromLTRB(this.left, this.top, this.right, this.bottom)
+    : assert(left != null),
+      assert(top != null),
+      assert(right != null),
+      assert(bottom != null);
```

Comment:
Same as last time, remember to mark the `const` linter warnings as ignore until we've rolled internally and embedders have been updated.